opengrok-indexer/src/main/java/org/opengrok/indexer/web/SearchHelper.java

```diff
@@ -344,8 +346,11 @@ public SearchHelper prepareExec(SortedSet<String> projects) {
         } catch (ParseException e) {
             errorMsg = PARSE_ERROR_MSG + e.getMessage();
         } catch (FileNotFoundException e) {
-//          errorMsg = "Index database(s) not found: " + e.getMessage();
-            errorMsg = "Index database(s) not found.";
+            errorMsg = "Index database not found. Check the index";
+            if (projects.size() > 0) {
+                errorMsg = errorMsg + " for projects: " + String.join(", ", projects);
+            }
+            errorMsg = errorMsg + " " + e.getMessage();
```

Comment:
Maybe use different separator instead of space? E.g. `.`, `;`. I think it will improve the readability a bit.

Reply:
good idea